src/chat/model_selector.rs

```diff
@@ -320,55 +320,72 @@ impl ModelSelector {
     }
 
     fn update_selected_model_info(&mut self, cx: &mut Cx, store: &Store) {
+        let is_loading = store.chats.model_loader.is_loading();
```

Comment:
I'm personally prefer how the code was organized before these changes, because it is easier to spot how the UI looks in each case.

Since this refactor was not related with this PR, can we just revert it?